src/session/user.rs

```diff
@@ -146,12 +147,88 @@ mod imp {
 
             let avatar = obj.avatar();
             let user_expression = gtk::ConstantExpression::new(obj);
-            let full_name_expression = super::User::full_name_expression(&user_expression);
-            full_name_expression.bind(avatar, "display-name", gtk::NONE_WIDGET);
+            let display_name_expression = super::User::display_name_expression(&user_expression);
+            display_name_expression.bind(avatar, "display-name", gtk::NONE_WIDGET);
         }
     }
 }
 
+fn last_seen_time_ago(unix: i32) -> String {
+    let datetime_now = glib::DateTime::new_now_local().unwrap();
+    let unix: i64 = unix.into();
+    let last_seen = glib::DateTime::from_unix_utc(unix)
+        .unwrap()
+        .to_local()
+        .unwrap();
+
+    let seconds = datetime_now.difference(&last_seen) / 1_000_000;
+    let minutes = seconds / 60;
+    let hours = minutes / 60;
+    if minutes < 120 {
+        if minutes == 0 {
+            gettext("last seen just now")
+        } else if minutes == 1 {
+            gettext("last seen a minute ago")
+        } else if minutes < 60 {
+            gettext!("last seen {} minutes ago", minutes)
+
+        // greater or equal to 60 minutes
+        } else {
+            gettext("last seen an hour ago")
+        }
+    } else if (
```

Comment:
This can be simplified
```suggestion
    if minutes == 0 {
        gettext("last seen just now")
    } else if minutes == 1 {
        gettext("last seen a minute ago")
    } else if minutes < 60 {
        gettext!("last seen {} minutes ago", minutes)

        // greater or equal to 60 minutes
    } else if minutes < 120 {
        gettext("last seen an hour ago")
    } else if (
```